playgrounds/petclinic-maven-sb-2/src/main/resources/db/postgres/schema.sql

```
@@ -0,0 +1,52 @@
+CREATE TABLE IF NOT EXISTS vets (
+  id         INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+  first_name TEXT,
+  last_name  TEXT
+);
+CREATE INDEX ON vets (last_name);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)